pandas/core/frame.py

```diff
@@ -4628,6 +4629,31 @@ def unstack(self, level=-1, fill_value=None):
     1      b          B          E      3
     2      c          B          E      5
 
+    .. versionadded:: 0.22.0
+
+    Simultaneously melt multiple groups of columns:
+
+    >>> df2 = pd.DataFrame({'City': ['Houston', 'Miami'],
```

Comment:
simple examples first

Reply:
This is as simple as it gets. One id column, two column groups of each length two and only two rows of data.